clients/benchmarks/client.cpp

```diff
@@ -398,6 +398,10 @@ try
          bool_switch(&arg.scaleD_vector)->default_value(false),
          "Apply scaleD vector")
 
+        ("grouped_gemm",
```

Comment:
Please update README.md of becnhmarks as well

Reply:
done